### PR TITLE
Add support for effect counter labels and formula badges

### PR DESCRIPTION
**Pull request by CarlosFdez**
Labels exist in data yet but they're not shown. We can probably add it to the tooltip under the name?

image

image

image

### CODE REVIEW

**Comment by CarlosFdez** [comment]
I can add a devise a strategem implementation as part of this or separately, but functioning it looks like this

https://user-images.githubusercontent.com/1286721/215319326-0dd56869-d32e-4cf5-86c1-24cdb4e5b074.mp4

Note that the "value" badge type cannot be manually created, it only exists as a result of the formula evaluation.

File: static/lang/en.json [hunk 1]
```diff
@@ -2201,9 +2201,16 @@
...
+                "AddBadge": "Add",
+                "AddLabel": "Add Label",
+                "LabelPlaceholder": "Stage Label",
                 "BadgeType": {
-                    "Counter": "Counter"
+                    "counter": "Counter",
+                    "formula": "Formula",
+                    "value": "Value"
+                },
+                "ExtraSelections": {
```
Comment (CarlosFdez) [inline static/lang/en.json:2212]: This fundamentally bothers me and I'm willing to try a replacement.

Comment (stwlam) [reply static/lang/en.json:2226]: "Fixed value" communicates it a little better

Comment (CarlosFdez) [reply static/lang/en.json:2226]: For Mild/Moderate/Severe or is this about Value? I could buy it for Value.

My umbrage was at having an ExtraSelections section 🙃

Comment (stwlam) [reply static/lang/en.json:2226]: yeah what the fuck is that, and why is it bactrian'd

Comment (stwlam) [reply static/lang/en.json:2226]: Yes for value

**Pull request merged**